vignettes/epiparameter.Rmd

```diff
@@ -0,0 +1,521 @@
+---
+title: "Introduction to epiparameter"
+output: rmarkdown::html_vignette
+bibliography: references.bib
+csl: evolution.csl
+vignette: >
+  %\VignetteIndexEntry{epiparameter}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+This vignette will use a range of R packages for formatting data and performing a range of calculations. These packages can be thought of as either general data science tools, which will come from the [*tidyverse*](https://www.tidyverse.org/), or tools specific to epidemiological tasks.
+
+```{r setup}
+library(tibble)
+library(epiparameter)
+library(EpiEstim)
+library(EpiNow2)
+library(R0)
+library(outbreaks)
+library(incidence2)
+```
+
+`epiparameter` is an R package that contains a range of delay distributions collated from peer-reviewed reports and meta-analyses, as well as functions to extract the parameters of a right-skewed parametric probability distribution from its percentiles or a median and range. If you would like to contribute to the database on delay distributions in this R package see the [contributing section](#contributing) at the end of this document.
+
+Definitions:
```

Comment:
This still needs to be completed, right?

Reply:
I wasn't planning on adding anything else to this section. Is there something in particular you think needs adding?